src/slack_notify.ml

```diff
@@ -66,16 +70,22 @@ let execute token username channel icon_url icon_emoji msg =
   let channel = Slacko.channel_of_string channel in
   let chat = Slacko.Channel channel in
   let msg = Slacko.message_of_string msg in
+  let attachments =
```

Comment:
Maybe it would make sense to implement our own `Option.map` to avoid this boilerplat-y match.